cfpb/cfgov-refresh
# Cleanup deprecated mega menu-related blocks

This commit removes some obsolete StreamField block types and associated frontend code for modules associated with the old mega menu. None of these modules are used with the current version of the menu.

## How to test this PR

1. Backend and frontend tests confirm none of this is used.
2. `yarn run gulp build` and run a local server to confirm this doesn't affect the current mega menu.

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing with the feature/section you're addressing, e.g., "Mega Menu: fix layout bug" or "Paying for College: Update content on Repay tool"
  - [This repo’s docs](https://cfpb.github.io/cfgov-refresh/) (edit the files in the `/docs` folder) – for basic, close-to-the-code documentation on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
